packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_dex_poolswap.test.ts

```diff
@@ -160,75 +161,44 @@ describe('dex.poolswap()', () => {
     expect(prevouts[0].value.toNumber()).toBeGreaterThan(9.999)
   })
 
-  it('should fail poolSwap due to maxPrice lower than reserveA/reserveB', async () => {
-    await providers.randomizeEllipticPair()
-    await container.waitForWalletBalanceGTE(1)
-    const addressLP = await container.getNewAddress()
-    await addPoolLiquidity(container, {
-      tokenA: 'DFI',
-      amountA: 100,
-      tokenB: 'CAT',
-      amountB: 20,
-      shareAddress: addressLP
-    })
-    await providers.setupMocks()
-    await utxosToAccount(container, 100, { address: await providers.getAddress() })
-    await sendTokensToAddress(container, await providers.getAddress(), 2, 'CAT')
-    await fundEllipticPair(container, providers.ellipticPair, 10)
-    const script = await providers.elliptic.script()
-
-    const txn = await builder.dex.poolSwap({
-      fromScript: script,
-      fromTokenId: pairs.CAT.tokenA,
-      fromAmount: new BigNumber('2'),
-      toScript: script,
-      toTokenId: pairs.CAT.tokenB,
-      maxPrice: new BigNumber('0.00000003')
-    }, script)
-
-    const promise = sendTransaction(container, txn)
-    await expect(promise).rejects.toThrow(DeFiDRpcError)
-    await expect(promise).rejects.toThrow('Price is higher than indicated')
-  })
-
-  it('should pass with 5.0', async () => {
+  it('should pass with 500', async () => {
     providers.randomizeEllipticPair()
     await container.waitForWalletBalanceGTE(1)
     const addressLP = await container.getNewAddress()
     await addPoolLiquidity(container, {
       tokenA: 'DFI',
-      amountA: 100,
-      tokenB: 'BIRD',
-      amountB: 20,
+      amountA: 20,
+      tokenB: 'GOAT',
+      amountB: 10000,
       shareAddress: addressLP
     })
     await providers.setupMocks()
     await utxosToAccount(container, 100, { address: await providers.getAddress() })
-    await sendTokensToAddress(container, await providers.getAddress(), 2, 'BIRD')
+    await sendTokensToAddress(container, await providers.getAddress(), 2, 'GOAT')
     await fundEllipticPair(container, providers.ellipticPair, 10)
     const script = await providers.elliptic.script()
 
     const txn = await builder.dex.poolSwap({
       fromScript: script,
-      fromTokenId: pairs.BIRD.tokenA,
-      fromAmount: new BigNumber('2'),
+      fromTokenId: pairs.GOAT.tokenB,
```

Comment:
Change multiple price related tests from `convert DFI to other` to `convert other to DFI`, minting other token is way faster than wait for DFI token balance.

Large token volume reduce the slippage effect and improve the test accuracy